Branch-SDK-TestBed/AndroidManifest.xml

```diff
@@ -31,18 +31,35 @@
             	<category android:name="android.intent.category.BROWSABLE" />
         	</intent-filter>
         </activity>
-        <activity
-            android:name="io.branch.branchandroiddemo.CreditHistoryActivity">
-        </activity>
-        <activity
-            android:name="io.branch.branchandroiddemo.ReferralCodeActivity">
+
+        <activity android:name="io.branch.branchandroiddemo.CreditHistoryActivity"/>
+
+        <activity android:name="io.branch.branchandroiddemo.ReferralCodeActivity"/>
+
+        <activity android:name=".AutoDeepLinkTestActivity">
+            <!-- Keys for auto deep linking this activity -->
+            <meta-data android:name="io.branch.sdk.auto_link_keys" android:value="auto_deeplink_key_1,auto_deeplink_key_2" />
+            <!-- Optional request ID for launching this activity on auto deep link key matches -->
+            <meta-data android:name="io.branch.sdk.auto_link_request_code" android:value="201" />
```

Comment:
Ah I see - so their main activity would call onActivityResult and pass in that code?